code/modules/power/apc.dm

```diff
@@ -811,13 +840,13 @@
 		return 0
 	if ( ! (istype(user, /mob/living/carbon/human) || \
 			istype(user, /mob/living/silicon)))
-		user << "\red You don't have the dexterity to use this [src]!"
+		user << "<span class='warning'>You don't have the dexterity to use this [src]!</span>"
 		return 0
 	if(user.restrained())
-		user << "\red You must have free hands to use this [src]"
+		user << "<span class='warning'>You must have free hands to use this [src]</span>"
 		return 0
 	if(user.lying)
-		user << "\red You must stand to use this [src]!"
+		user << "<span class='warning'>You must stand to use this [src]!</span>"

```

Comment:
In this case, warning is fine. But if you want it to actually match what \red does, use class='alert'